static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -141,7 +145,7 @@ const handlePlay = (atomId: string, player: AtomPlayer): void => {
         mainMedia.classList.add('atom-playing');
     }
 
-    if (overlay && endSlate) {
+    if (overlay && endSlate && !endSlate.rendered) {
```

Comment:
This is a fix, we don't want to call `endSlate.fetch` if it's been rendered already.